recommended.code-workspace

```diff
@@ -63,7 +63,9 @@
       "redhat.vscode-yaml",
       "silvenon.mdx",
       "jounqin.vscode-mdx",
-      "errata-ai.vale-server"
+      "errata-ai.vale-server",
+      "bierner.markdown-mermaid",
+      "bpruitt-goddard.mermaid-markdown-syntax-highlighting"
```

Comment:
When applying this to the content repo, please also add these to the recommended plugins in the content repo.